### ISSUE TITLE
Description list stays open after attempting to type in a different text input

### ISSUE THREAD

**Issue by plaporte** [FaridSafi/react-native-google-places-autocomplete#566]
After selecting an item in from the google places description list, it will close, however when moving to the next textInput and typing, the description list opens back up. Here's a gif:

![ezgif com-resize](https://user-images.githubusercontent.com/23533193/82499821-f77bcd80-9ab7-11ea-8b09-d9e4e2f97ca5.gif)

#### Expected behavior
I expect it to close until I tap back into the google places text input.

#### Code Example (required - issue will be closed without this)

My Code:
```
<GooglePlacesAutocomplete
                    placeholder="Search location"
                    // fetchDetails={true}
                    onPress={(data, details = null) => {
                      setFieldValue("location", data.description);
                    }}
                    query={{
                      key: GOOGLE_SEARCH_API,
                      language: "en",
                    }}
                    autoCapitalize={false}
                    autoCorrect={false}
                    placeholderTextColor={Global.accentColor2}
                    styles={googleSearchBox}
                    autoFocus={false}
           />
```

```
  const googleSearchBox = {
    container: {
      backgroundColor: Global.mainBackgroundColor,
      marginHorizontal: 0,
      padding: 0,
      marginVertical: 10,
      border: 0,
      borderRadius: 15,
    },
    description: {
      color: Global.mainFontColor,
      fontSize: 16,
      paddingright: 10,
    },
    textInputContainer: {
      borderBottomWidth: 3,
      paddingLeft: 10,
      borderBottomColor: Global.mainColor,
      fontSize: 18,
      backgroundColor: Global.accentColor1,
      marginLeft: 0,
      color: Global.mainFontColor,
      borderTopWidth: 0,
      borderRadius: 15,
    },
    textInput: {
      fontSize: 18,
      backgroundColor: "transparent",
      color: Global.mainFontColor,
      padding: 0,
      paddingBottom: 10,
      marginLeft: 0,
    },
    predefinedPlacesDescription: {
      // color: Global.mainColor,
    },
    powered: {
      display: "none",
      backgroundColor: "transparent",
    },
    poweredContainer: {
      display: "none",
      backgroundColor: "transparent",
    },
  };
```

#### Screenshots
GIF Above

#### Where is this bug happening

- [x] iOS
- [x] Android
- [ ] Web

If you are using expo please indicate here:

- [x] I am using expo

#### Version Information (please complete the following information)

- Library Version: [e.g. 1.4.2]
    "react-native-google-places-autocomplete": "^1.6.1",

- React Native Version: [e.g. 0.62.2]
    "expo": "~37.0.3",
    "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz",

**Comment by bell-steven** [comment]
Thanks for reporting this. Would you mind provided a full code reproduction of the issue? In the meantime, the `listViewDiplayed` prop might help you out here.

**Comment by bell-steven** [comment]
I was able to reproduce this.  There is actually a PR open already that should solve this. #398

**Comment by bell-steven** [comment]
Fixed in v1.7.1
